docs/source/create_channel/create_channel_participation.md

```diff
@@ -200,7 +200,7 @@ OrdererEndpoints:
 
 #### Orderer:
 
-- **Orderer.OrdererType** Set this value to `etcdraft`. As mentioned before, this process does not work with Solo or Kafka ordering nodes.
+- **Orderer.OrdererType** Set this value to `etcdraft`. As mentioned before, this process does not work with Solo ordering nodes.
```

Comment:
Incidentally - Solo has also been removed, so maybe we could also remove Solo?

Reply:
Since in this PR we remove only Kafka, I didn't touch places were Solo appeared.
I suggest doing that in a separate PR.

Reply:
makes sense - do you want to go ahead and do it, or do you want to open an issue so we don't forget?

Reply:
@ale-linux @MayRosenbaum There is an issue on solo #3514 , it contains some open subtasks to do some cleanup